backend/invertedIndex.js

```diff
@@ -0,0 +1,38 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+const tokenizeAndNormalize = (text) => {
+  return text
+    .toLowerCase()
+    .replace(/[^a-z0-9\s]/g, "")
```

Comment:
is this needed? If the business has unicode characters such as chinese characters, the user will not be able to search for that business using those characters (because we're stripping out all non alphanumeric characters).

Reply:
Thank you. I've added this